tests/test_cli.py

```diff
@@ -129,7 +129,7 @@ def test_that_lyrics_calls_requests(self, f_get_lyrics, mock_requests):
 		Test that get_lyrics calls requests
 		"""
 		self.assertEqual(lyrics(
-			"Pixel2XL", "Elgoog", True), "Couldn't get lyrics for Pixel2XL by Elgoog.\nPhone is dope")
+			"Pixel2XL!", "Elgoog", True), "Couldn't get lyrics for Pixel2XL! by Elgoog.\nPhone is dope")
```

Comment:
if you change this test to pass then you should add one for the changes you're making

Reply:
oops... forgot to click push origin